packages/block-library/src/tag-cloud/edit.js

```diff
@@ -82,6 +96,54 @@ function TagCloudEdit( { attributes, setAttributes, taxonomies } ) {
 					max={ MAX_TAGS }
 					required
 				/>
+				<Flex>
+					<FlexItem isBlock>
+						<UnitControl
+							label={ __( 'Smallest size' ) }
+							value={ smallestFontSize }
+							onChange={ ( value ) => {
+								const unit = value.slice( -2 );
+								if ( largestFontSize.indexOf( unit ) === -1 ) {
+									setAttributes( {
+										largestFontSize:
+											largestFontSize.slice( 0, -2 ) +
+											unit,
+									} );
+								}
+								value = value.slice( 0, -2 );
+								const newValue =
+									value && +value >= 8 ? +value : 8;
+								setAttributes( {
```

Comment:
The passed new value is already a string with the unit attached, so there is no need to parse and reconstruct for the main attribute. In this case `smallestFontSize` for example.

Reply:
I have introduced new function `onFontChange` common for both `smallestFontSize` and `largestFontSize`. I hope this time I did make mistake! 😅